codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/credentialresolverservices/InternalModelIntegration.kt

```diff
@@ -0,0 +1,43 @@
+package software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices
```

Comment:
The model integration that removes unused operations from internal service clients. For STS, we only need assumeRole and assumeRoleWithWebIdentity. Reduces runtime module size.